integration_tests/e2e/functionalSkills/functionalSkills.cy.ts

```diff
@@ -7,11 +7,13 @@ context(`Display a prisoner's functional skills`, () => {
   beforeEach(() => {
     cy.task('reset')
     cy.task('stubSignInAsReadOnlyUser')
+    cy.task('stubGetAllPrisons')
```

Comment:
The first 44 files changed are cypress test changes where existing tests are being changed to use the new /learnerAssessments stub rather than the previous /learnerProfile stub
The only notable exception is this cypress test for the main Functional Skills page - it needs to call the new /learnerAssessments stub (to get any assessments recorded in Curious 2 from the v2 property), but also it needs to  call the old /learnerProfile stub to get any assessments recorded in Curious 1